packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.ts

```diff
@@ -191,6 +193,11 @@ const getAllProps = (
     borderRadius: getBorderRadiusValue(props.borderRadius, props.theme, size),
     transform: getResponsiveValue(props.transform, size),
     lineHeight: getSpacingValue(props.lineHeight, props.theme, size),
+    border: getResponsiveValue(props.border, size),
+    borderTop: getResponsiveValue(props.borderTop, size),
+    borderRight: getResponsiveValue(props.borderRight, size),
+    borderBottom: getResponsiveValue(props.borderBottom, size),
+    borderLeft: getResponsiveValue(props.borderLeft, size),
```

Comment:
These are for `BaseBox` only. Had few requirements while using BaseBox in storybook